package.json

```diff
@@ -33,21 +33,23 @@
     "url": "https://github.com/ssense/vue-carousel/issues"
   },
   "homepage": "https://github.com/ssense/vue-carousel#readme",
+  "dependencies": {
+    "@babel/polyfill": "^7.0.0-beta.53",
```

Comment:
You can update it to stable 7.0.0

Reply:
Yeah, the stable version it was just released a few days ago lol a sign I've let this PR sit here for too long